check/test_test.go

```diff
@@ -690,4 +703,14 @@ func TestToNumeric(t *testing.T) {
 			t.Errorf("TestToNumeric - Expected to return %d,%d , but instead got %d,%d", 5, 6, f, s)
 		}
 	}
+	for _, c := range cases {
+		f, s, err := toNumericOctal(c.firstValue, c.secondValue)
+		if c.expectedToFail && err == nil {
```

Comment:
I think it's a best practice for **Unit tests** to test only one function at a time, so the test for `toNumericOctal` belongs on its own Test function **`TestToNumericOctal`**.

Reply:
Or even better, put the bitmask in octal representation (Yehuda’s suggestion in our tram meeting) and then you wouldn’t need a conversion function